packages/component/src/Styles/defaultStyleOptions.js

```diff
@@ -49,6 +49,9 @@ const DEFAULT_OPTIONS = {
   // Markdown
   markdownRespectCRLF: true,
 
+  // Rich Cards
+  richCardsWrapTitle: false,
```

Comment:
I think we should name it `richCardWrapTitle`.

Could you put a comment here and say it will also applies to subtitle?